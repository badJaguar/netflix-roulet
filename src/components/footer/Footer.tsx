import * as React from "react";

import { icon } from "../../assets/icons";
import { ErrorBoundry } from "../../services/ErrorBoundry";

export class Footer extends React.Component {
    public render() {
        return (
            <ErrorBoundry>
                <footer>
                    <icon.logo />
                </footer>
            </ErrorBoundry>
        );
    }
}
