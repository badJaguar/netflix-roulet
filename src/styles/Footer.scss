@import './colors.scss';

footer {
    position: unset;
    left: 0;
    bottom: 0;
    width: 100%;
    text-align: center;
    background-color: $gray;
    margin-top: 70px;
    padding-top: 10px;
}