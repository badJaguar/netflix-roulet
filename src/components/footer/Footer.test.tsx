import * as React from "react";

import { shallow } from "enzyme";
import { Footer } from "./Footer";

describe("<Footer />", () => {

    it("renders Footer element as expected", () => {
        const wrapper = shallow(<Footer />);
        expect(wrapper).toMatchSnapshot();
    });
});
