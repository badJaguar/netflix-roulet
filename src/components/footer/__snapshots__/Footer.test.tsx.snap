// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders Footer element as expected 1`] = `
<ErrorBoundry>
  <footer>
    <Component />
  </footer>
</ErrorBoundry>
`;
